app/src/main/java/org/sopt/and/ui/navigation/NavGraph.kt

```diff
@@ -0,0 +1,67 @@
+package org.sopt.and.ui.navigation
+
+import androidx.compose.runtime.Composable
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.toRoute
+import com.google.accompanist.permissions.ExperimentalPermissionsApi
+import org.sopt.and.model.Route
+import org.sopt.and.ui.home.HomeScreen
+import org.sopt.and.ui.login.LogInScreen
+import org.sopt.and.ui.myprofile.MyProfileScreen
+import org.sopt.and.ui.search.SearchScreen
+import org.sopt.and.ui.signup.SignUpScreen
+import org.sopt.and.ui.signup.SignUpState
+
+
+@OptIn(ExperimentalPermissionsApi::class)
+@Composable
+fun NavGraph(navController: NavHostController, bottomBarVisible: (Boolean) -> Unit) {
+    NavHost(navController = navController, startDestination = Route.LogIn("", "")) {
```

Comment:
개인 취향이긴한데 줄바꿈 하는것도 가독성을 위한 하나의 방법이라 생각합니다!!
```suggestion
fun NavGraph(
navController: NavHostController,
bottomBarVisible: (Boolean) -> Unit
) {
    NavHost(
        navController = navController,
        startDestination = Route.LogIn("", "")
    ) { ~~
```

Reply:
줄 바꿈 신경써 보겠습니다 모두의 눈을위해..